tests/integrations/raftstore/test_merge.rs

```diff
@@ -1037,3 +1036,50 @@ fn test_merge_isloated_not_in_merge_learner() {
     // We can see if the old peer 2 is destroyed
     must_get_equal(&cluster.get_engine(2), b"k123", b"v123");
 }
+
+// Test if a learner can be destroyed properly when it's isloated and removed by conf change
+// before another region merge to its region
+#[test]
+fn test_merge_isloated_stale_learner() {
```

Comment:
Merge doesn't seem to be necessary in this case.

Reply:
Because it's a learner. Merge and then split are used to trigger a learner to send a wake-up msg.